client.lua

```diff
@@ -80,13 +84,31 @@ RegisterCommand("drugs-weed", function(source, args, rawCommand)
 	end
 end)
 
-RegisterCommand("drugs-coke", function(source, args, rawCommand)
+AddEventHandler('baseevents:onPlayerDied', function()
+	if high == true then
+	StopGameplayCamShaking(true)
+		resetAnims()
+		DisabledRun = false
+		high = false
+		SetTransitionTimecycleModifier('default', 0.35)
+		--SetPedMoveRateOverride(source, 0)
+		SetRunSprintMultiplierForPlayer(source, 1.01)
+		SetEntityMaxHealth(GetPlayerPed(-1), 200)
+	TriggerEvent('chat:addMessage', {
+		color = { 255, 0, 0},
+		multiline = true,
+		args = {"WadL Drugs", "^2^*You have died under the influence"}
+	})
+end
+end)
+

```

Comment:
Death Handler, When you die with drugs activated it will say you have died under the influence, I believe overdosing still says you overdose with that message.

Reply:
Would suggest removing "WadL Drugs" and editing it to be "Drugs" or "System"